packages/shared/src/contexts/ActiveFeedContext.tsx

```diff
@@ -1,7 +1,9 @@
 import { createContext } from 'react';
 import { FeedReturnType } from '../hooks/useFeed';
+import { AllFeedPages } from '../lib/query';
 
 export type ActiveFeedContextValue = {
+  feedName?: AllFeedPages;
```

Comment:
We don't need this one since we are using `ActiveFeedNameContext`?

Reply:
I think you looked at non refreshed one as it was removed :)